testing/itest/src/infra/subnet.rs

```diff
@@ -305,10 +308,16 @@ impl SubnetNode {
 
         let subnet_id = self.subnet_id_cli_string();
 
-        let node_std_out =
-            File::create(format!("./{subnet_id:}_node_{:}.log", self.node.tcp_port))?;
-        let node_std_err =
-            File::create(format!("./{subnet_id:}_node_{:}.err", self.node.tcp_port))?;
+        let base_path = Path::new(DEFAULT_LOG_DIR);
+        fs::create_dir_all(base_path).unwrap();
```

Comment:
maybe we should use `?` here instead of `unwrap`.

Reply:
Fixed